src/GovUk.Education.ExploreEducationStatistics.Common/Requests/QueryRequests.cs

```diff
@@ -0,0 +1,90 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using FluentValidation;
+using GovUk.Education.ExploreEducationStatistics.Common.Model.Data.Query;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Requests;
+
+public record FullTableQueryRequest
+{
+    public Guid SubjectId { get; set; }
+
+    public List<Guid> LocationIds { get; set; } = new();
+
+    public TimePeriodQuery? TimePeriod { get; set; }
+
+    public IEnumerable<Guid> Filters { get; set; } = new List<Guid>();
```

Comment:
Should this be nullable if it's optional, or is it important for it to always be a list, even if it's an empty one?

Reply:
This is the way it was handled before my changes. I think you've got a point here, but it might be a bit of an involved change - I would probably have to change the frontend as well, and it could be nullable in several instances - for what I consider very little benefit.